src/test/java/enmasse/kafka/bridge/KafkaTestBase.java

```diff
@@ -0,0 +1,77 @@
+package enmasse.kafka.bridge;
```

Comment:
I guess that this PR should be useful only for adding the possibility to run tests without the need of an external Kafka cluster right ? So this class should not be needed. We don't use it in the tests we already have in the bridge. For now we need the Before and After methods for starting the embedded Kafka cluster. You could get rid of the "extends KafkaTestBase" in the previous KafkaClusterTestBase. It's possible that we'll re-add it when moving to use the Vert.x Kafka client, but let's do step by step. wdyt ?